examples/atari_deepRL_saliency.ipynb

```diff
@@ -600,15 +582,9 @@
     "    for img_idx in range(start_frame, end_frame+1):\n",
     "        sal_idx = img_idx-start_frame\n",
     "        \n",
-    "        # Post-processing here follows Greydanus et al., '18\n",
-    "        img = history['ins'][img_idx].copy().astype('uint16')\n",
-    "        \n",
-    "        # Highlight actor saliency in blue and critic saliency in red\n",
-    "        img[35:195,:,2] += postprocess_sal(actor_sal_maps[sal_idx], 300).astype('uint16')\n",
-    "        img[35:195,:,0] += postprocess_sal(critic_sal_maps[sal_idx], 600).astype('uint16')\n",
+    "        frame = visualize_saliency(history['ins'][img_idx], actor_sal_maps[sal_idx], critic_sal_maps[sal_idx])\n",
```

Comment:
Just to confirm, do we need to create a copy of this frame history, or is it OK to pass in the original input here? We seem to create a copy when visualizing the individual frame, but not when generating the video.

e.g. using `history['ins'][img_idx].copy()`

Reply:
I believe the copy is not needed. The image is not mutated by the `visualize_saliency()` function. The `cv2.addWeighted()` function used there returns a new image.